Readme.md

```diff
@@ -1,94 +1,87 @@
+Note that [Entity Framework](https://learn.microsoft.com/en-us/ef/ef6/) includes the corresponding functionality out-of-the-box if you define the [Timestamp](https://learn.microsoft.com/en-us/ef/ef6/modeling/code-first/data-annotations#timestamp) field in your data model:
 
 ```cs
-        [Timestamp]
-        public Byte[] RowVersion { get; set; }
+[Timestamp]
+public Byte[] RowVersion { get; set; }
 ```
 
-<p> </p><p>Check the corresponding section in the <a href="http://msdn.microsoft.com/en-us/data/jj591583.aspx"><u>Code First Data Annotations</u></a> to learn more on this subject. Once you define this field, entity framework will automatically perform concurrency checks when saving data to the database. The <a href="http://msdn.microsoft.com/en-us/library/system.data.entity.infrastructure.dbupdateconcurrencyexception.aspx"><u>DbUpdateConcurrencyException</u></a> will be thrown in case of concurrency conflict.</p><p>This scenario is quite easy to implement in stateful environments like WinForms. But in the case of a stateless ASP.NET MVC environment, you need some additional logic over the default Entity Framework's logic to keep the current <strong>RowVersion</strong> value for the specific object between postbacks. In a regular MVC project, it is possible to use a hidden field for this purpose (e.g., see <a href="http://www.asp.net/mvc/tutorials/getting-started-with-ef-5-using-mvc-4/handling-concurrency-with-the-entity-framework-in-an-asp-net-mvc-application"><u>Handling Concurrency with the Entity Framework in an ASP.NET MVC Application (7 of 10)</u></a>). In case of our MVC extensions you can use the approach illustrated in the <a href="http://documentation.devexpress.com/#AspNet/CustomDocument11816"><u>How to: Access Server Data on the Client Side</u></a> document to pass RowVersion-related data on the client side:</p>
-
-```cs
-    settings.CustomJSProperties = (s, e) => {
-        MVCxGridView gridView = (MVCxGridView)s;
-        
-        var dictionary = new System.Collections.Generic.Dictionary<object, string>();
-
-        for (int i = 0; i < gridView.SettingsPager.PageSize; i++) {
-            var visibleIndex = i + gridView.VisibleStartIndex;
-            if (visibleIndex >= gridView.VisibleRowCount)
-                break;
-
-            object[] rowValues = (object[])gridView.GetRowValues(visibleIndex, gridView.KeyFieldName, "RowVersion");
-
-            dictionary[rowValues[0].ToString()] = Convert.ToBase64String((byte[])rowValues[1]);
-        }
+Once you define the `Timestamp` field, entity framework automatically performs concurrency checks when saving data to the data base. In case of concurrency conflict, the [DbUpdateConcurrencyException](https://learn.microsoft.com/en-us/dotnet/api/system.data.entity.infrastructure.dbupdateconcurrencyexception) will be thrown.
```

Comment:
```suggestion
Once you define the `Timestamp` field, entity framework automatically performs concurrency checks when saving data to the database. In case of concurrency conflict, the [DbUpdateConcurrencyException](https://learn.microsoft.com/en-us/dotnet/api/system.data.entity.infrastructure.dbupdateconcurrencyexception) is thrown.
```